tests/integration/cgroups.bats

```diff
@@ -119,6 +119,86 @@ function setup() {
 	#
 }
 
+@test "runc exec (cgroup v2, ro cgroupfs, new cgroupns) does not chown cgroup" {
+	requires root cgroups_v2 systemd
+
+	set_cgroups_path
+
+	# configure a user namespace
+	update_config '.linux.namespaces += [{"type": "user"}]'
+	update_config '.linux.uidMappings += [{"hostID": 100000, "containerID": 0, "size": 65536}]'
+	update_config '.linux.gidMappings += [{"hostID": 100000, "containerID": 0, "size": 65536}]'
```

Comment:
A single update_config statement is enough. Not important, but we tend to use a single one elsewhere.